docs/proposals/rename-group.md

```diff
@@ -0,0 +1,165 @@
+# Allocation strategies
+
+## Summary
+
+Change `topolvm.cybozu.com` used in the CRD and plugin name of TopoLVM to `topolvm.io`.
+
+## Motivation
+
+We are thinking of donating the TopoLVM project to CNCF in the future.
+So we would like to remove the domain name "cybozu.com," which contains the specific company's name, from the project.
+
+### Goals
+
+- Change `topolvm.cybozu.com` used in TopoLVM's CRD and plugin name to `topolvm.io`
+- Keep TopoLVM available at `topolvm.cybozu.com` for existing users(but does not support concurrent use with topolvm.io partly)
```

Comment:
```suggestion
- Keep TopoLVM available at `topolvm.cybozu.com` for existing users (but does not support concurrent use with topolvm.io partly)
```

Reply:
Also, I could not understand what you tried to mean by saying "partly".

Reply:
Sorry, I fixed this description. Would you review this again?